modes/company/evil-collection-company.el

```diff
@@ -69,26 +69,19 @@ be set through custom or before evil-collection loads."
 ;;;###autoload
 (defun evil-collection-company-setup ()
   "Set up `evil' bindings for `company'."
-  (evil-collection-define-key nil 'company-active-map
-    (kbd "C-n") 'company-select-next-or-abort
-    (kbd "C-p") 'company-select-previous-or-abort
-    (kbd "C-j") 'company-select-next-or-abort
-    (kbd "C-k") 'company-select-previous-or-abort
-    (kbd "M-j") 'company-select-next
```

Comment:
Ran through this fast and not with an emacs-Q setup but some initial thoughts.
C-n/C-p worked for me but the other keys didn't. I bet I have some layer that binds these keys so doing this unbinding drops evil's key priority into some other mode.

[evil-collection] --> [some random potentially uncontrolled layer that binds random keys] --> [emacs state bindings] 

is my guess at the priority. If we focus on only unbinding keys, we'll run into issues where something else have overwritten the emacs bindings.